arangod/Replication/Syncer.cpp

```diff
@@ -311,15 +311,14 @@ std::string Syncer::getCName(VPackSlice const& slice) const {
 LogicalCollection* Syncer::getCollectionByIdOrName(TRI_vocbase_t* vocbase,
                                                    TRI_voc_cid_t cid,
                                                    std::string const& name) {
-  
-  arangodb::LogicalCollection* idCol = vocbase->lookupCollection(cid);
+  auto* idCol = vocbase->lookupCollection(cid).get();
   arangodb::LogicalCollection* nameCol = nullptr;
```

Comment:
maybe transform this to a shared_ptr as well ?

Reply:
I tried to keep the changes to a minimum otherwise there are too many files in the changeset
although I do agree that eventually raw pointers should be shared pointers